guides/plugins/plugins/index.md

```diff
@@ -29,3 +46,9 @@ The detection of static plugins is not done via the Shopware administration. The
 # You can find the vendor/package name in the plugin's composer.json file under "name"
 composer req <vendor>/<plugin-name>
 ```
+
+### Symfony Bundle / Shopware Bundle
+
+It's also possible to use Shopware/Symfony bundles instead of plugins. This is useful if you don't want to have the lifecycle of plugins and don't want it manageable via the Shopware administration. The bundles are typically installed via composer and are not managed by the Shopware administration.
```

Comment:
```suggestion
It's also possible to use Shopware/Symfony bundles instead of plugins.
This is useful if you don't want to have the lifecycle of plugins and don't want it manageable via the Shopware administration.
The bundles are typically installed via composer and are not managed by the Shopware administration.
```